### PR TITLE
Instructions to donwload & add assessment items post-install  - fixes #3950

### LINKED ISSUE #3950: Document the download and installation instructions for assesment.zip
I spent some time searching for this the other day and found the command inside the HipChat history, which is not available outside our "family"...

There is a mention in 0.13 Release notes, but it gives no details on how to deal with it if you didn't do it during the installation process. I just added the download & installation instruction into the [Getting started wiki](https://github.com/learningequality/ka-lite/wiki/Getting-started), and I leave it to you to decide if & where should it go in the sphinx docs dev docs...

### PR DESCRIPTION
**Pull request by radinamatic**
Didn't build locally, but looks OK by Github.

Btw, I checked out the 0.13.x branch to add the same info to [FAQ](https://github.com/learningequality/ka-lite/blob/0.13.x/sphinx-docs/faq.rst), but the sphinx-docs seems empty... Where do I grab the faq.rst?

### CODE REVIEW

File: sphinx-docs/usermanual/userman_admin.rst [hunk 1]
```diff
@@ -490,6 +490,13 @@ These are resized videos. All in all, this will take around 23 GB of space.
 
 .. _BitTorrent Sync: http://www.getsync.com/
 
+Adding assessment items (exercises)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If you skipped the step for adding assessment items during install, run the following command (beware it can take a while, zip size is over 400Mb):
```
Comment (aronasorman) [inline sphinx-docs/usermanual/userman_admin.rst]: "The download size is over 500 MB".

File: sphinx-docs/usermanual/userman_admin.rst [hunk 2]
```diff
@@ -490,6 +490,13 @@ These are resized videos. All in all, this will take around 23 GB of space.
...
+
+``./manage.py unpack_assessment_zip -f https://learningequality.org/downloads/ka-lite/0.14/content/assessment.zip``
```
Comment (aronasorman) [inline sphinx-docs/usermanual/userman_admin.rst]: `kalite manage unpack_assessment_zip -f https://learningequality.org/downloads/ka-lite/0.14/content/assessment.zip`

File: sphinx-docs/usermanual/userman_admin.rst [hunk 3]
```diff
@@ -490,6 +490,13 @@ These are resized videos. All in all, this will take around 23 GB of space.
...
+
+You can alternatively download the zip file from https://learningequality.org/downloads/ka-lite/0.14/content/assessment.zip and unpack it locally.
```
Comment (aronasorman) [inline sphinx-docs/usermanual/userman_admin.rst]: > As an alternative, you can download the zip file from https://learningequality.org/downloads/ka-lite/0.14/content/assessment.zip and run the command on your local copy. This is useful if you're deploying KA Lite on multiple computers.

File: sphinx-docs/usermanual/userman_admin.rst (hunk 1 above)
Comment (radinamatic) [inline sphinx-docs/usermanual/userman_admin.rst]: Yikes, I think it was 400 just the other day! Mother must be proud... :P

**Comment by radinamatic** [comment]
Disregard the FAQ question, I figured out where the file is on 0.14 docs.

**Comment by aronasorman** [comment]
Other than those corrections, the changes look good!

However, the amount of merge commits is irking me :) This article has a good section on how to rewrite git's history: https://www.reviewboard.org/docs/codebase/dev/git/clean-commits/#rewriting-history

Try reducing the number of merge commits you have to 1 (or even 0!). After that, you can force push to this branch and it should have the new history.

**Comment by radinamatic** [comment]
You and me both (irked, specially if I have to meddle with VIM). I did the rebase:

```
$ git rebase -i develop
Successfully rebased and updated refs/heads/assesmentzip_to_docs.

$ git push -f origin assesmentzip_to_docs
Everything up-to-date
```

But all the merge commits are still there... :(

**Comment by aronasorman** [comment]
@radinamatic you have a cleaned up PR, right? Feel free to close this if so.

**Comment by radinamatic** [comment]
For some reason all the assessments that I installed the other day vanished so I was testing this instruction with `kalite manage unpack_assessment_zip -f https://learningequality.org/downloads/ka-lite/0.14/content/assessment.zip`, and the this error popped up:

`CommandError: We expect only one argument; the location of the zip.` 

When I removed the `-f` argument, no errors were reported, and after a while `Unpacking...` finished but I'm still not able to see the exercises. At this point I'm not sure if it's a problem on my machine or what, but for the purpose of this PR, I just wanted to check if the command should be amended  to exclude the `-f` argument.

**Comment by aronasorman** [comment]
@radinamatic you can try putting the `-f` after the zip location.

**Comment by radinamatic** [comment]
This PR is also cleaned (I'm actually getting hand on this VIM-thing ;), and it could be merged.

**Pull request merged**